toolbox/util/commonUtils.go

```diff
@@ -36,6 +36,9 @@ func ReadFile(filePath string) (string, error) {
 
 // WriteFile overwrites the file on the given path with content
 func WriteFile(filePath, content string) error {
+	if len(content) == 0 || content[len(content)-1] != '\n' {
```

Comment:
are you sure you want to change 0 bytes file (which is fine, sometimes marker files are like that) into 1 byte file for the new line ?

Reply:
This is a good point.